### DISCUSSION TITLE
Calendario de teams

### DISCUSSION THREAD

**Discussion by duR2** [Launch-X-Latam/MisionFrontEnd#7] [Q&A]
Se van a agriar las sesiones al calendario de Teams?

**Comment by Emilio17MC** [comment]
Las sesiones serán miércoles y viernes, y se publicará el link antes de la sesión en el canal de la misión. Sobre el calendario de Teams, me parece que no se van a mostrar desde ahí.

**Reply by LeonardoSoriano** [reply]
Disculpa, pero las sesiones empezarían mañana miércoles? o hasta 4 de octubre (la otra semana)?

**Reply by Cerecero** [reply]
Mañana.
